### ISSUE TITLE
Missing assembly when running AltCover.global in microsoft/dotnet:2.2-sdk Docker image

### ISSUE THREAD

**Issue by jonjomckay** [SteveGilham/altcover#74]
As of this morning, we're now getting the following error when running the `altcover` global tool in the `microsoft/dotnet:2.2-sdk` Docker image:

```
Error:
  An assembly specified in the application dependencies manifest (global-altcover.deps.json) was not found:
    package: 'System.Security.Principal.Windows', version: '4.6.0'
    path: 'runtimes/unix/lib/netcoreapp2.1/System.Security.Principal.Windows.dll'
```

You can reproduce it with the following:

1. `docker pull microsoft/dotnet:2.2-sdk`
2. `docker run -ti microsoft/dotnet:2.2-sdk /bin/bash`
3. (inside the container) `dotnet tool install --global AltCover.global`
4. (inside the container) `/root/.dotnet/tools/altcover`

I think this happened with the latest release from an hour ago? Love the project BTW - it's so valuable for us!

**Comment by jonjomckay** [comment]
Yes, pinning to `6.2.719` gets rid of the error for us, if that helps narrow it down.

**Comment by SteveGilham** [comment]
Believed fixed in new [release v6.2.727](https://www.nuget.org/packages/AltCover/6.2.727)

**Comment by jonjomckay** [comment]
Awesome - it's working again now, thank you!
